requirements/common.in

```diff
@@ -94,7 +94,7 @@ tornado
 orjson
 
 # Needed for iOS push notifications
-aioapns
+https://github.com/gnprice/aioapns/archive/2d69284481d328bd3e4168c631d75a38268eddb9.zip#egg=aioapns==3.1+git  # https://github.com/Fatal1ty/aioapns/pull/52
```

Comment:
We might want to move this fork into the `zulip` organization.

Reply:
Yeah that’d probably be good, but the more important consideration is that we should add a tag at that specific commit, so that the archive remains available even if upstream rebases the PR and GitHub garbage-collects the fork.

The existing such tags are https://github.com/zulip/talon/releases/tag/zulip-25522, https://github.com/andersk/zoneinfo/releases/tag/zulip-24573 (names corresponding to #25522, #24573 where they were introduced).

Reply:
Sure, good idea. And I'll follow that naming convention.